moveit_experimental/collision_distance_field/CMakeLists.txt

```diff
@@ -28,3 +23,11 @@ endif()
 add_library(collision_detector_hybrid_plugin src/collision_detector_hybrid_plugin_loader.cpp)
 target_link_libraries(collision_detector_hybrid_plugin ${catkin_LIBRARIES} ${MOVEIT_LIB_NAME})
 
+install(TARGETS ${MOVEIT_LIB_NAME} collision_detector_hybrid_plugin
+  LIBRARY DESTINATION lib)
+  
+install(DIRECTORY include/
+  DESTINATION ${CATKIN_GLOBAL_INCLUDE_DESTINATION})
```

Comment:
Do you know where the ${CATKIN_GLOBAL_INCLUDE_DESTINATION} place the header files? why not use just "include" instead?